AdyenCard/Utilities/BinLookup/BinLookupResponse.swift

```diff
@@ -0,0 +1,25 @@
+//
+// Copyright (c) 2020 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+internal struct BinLookupResponse: Response {
+    public var detectedBrands: [CardType]
+    
+    internal init(brands: [CardType]) {
+        self.detectedBrands = brands
+    }
+    
+    internal init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+        self.detectedBrands = (try container.decode([String].self, forKey: .detectedBrands)).compactMap { CardType(rawValue: $0) }
+    }
```

Comment:
do we still need to implement `init(from decoder: Decoder)` ourselves? making `CardType` `Decodable` doesn't do the trick?

Reply:
That was because of missing CodingKeys
👍